doc/source/advanced.rst

```diff
@@ -231,16 +228,31 @@ Advanced indexing with hierarchical index
 -----------------------------------------
 
 Syntactically integrating ``MultiIndex`` in advanced indexing with ``.loc`` is a
-bit challenging, but we've made every effort to do so. For example the
-following works as you would expect:
+bit challenging, but we've made every effort to do so. In general, MultiIndex
+keys take the form of tuples. For example, the following works as you would expect:
 
 .. ipython:: python
 
    df = df.T
    df
-   df.loc['bar']
    df.loc['bar', 'two']
 
+If you also want to index a specific column with ``.loc``, you have to use
+parentheses around the tuple like this:
+
+.. ipython:: python
+
+   df.loc[('bar', 'two'), 'A']
+
+You don't have to specify all levels of the ``MultiIndex`` by passing only the
+first elements of the tuple. For example, you can use this partially indexing to
+get all elements in the ``bar`` level as follows:
+
+df.loc['bar']
+
+This is identical to the slightly more verbose notation ``df.loc['bar',]`` using
```

Comment:
Maybe "This is a shortcut for the sligthly more verbose notation ``df.loc['bar',]`` (equivalent to ``df.loc[('bar',)]``)"